src/test/java/teammates/test/cases/browsertests/FeedbackContributionQuestionUiTest.java

```diff
@@ -123,6 +126,46 @@ public void testAddQuestionAction() throws Exception {
     }
 
     @Override
+    protected void testDestructiveChanges() {
+        // Create a dummy response
+        FeedbackQuestionAttributes question = BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 1);
+        FeedbackResponseAttributes fra = new FeedbackResponseAttributes(
+                                                feedbackSessionName,
+                                                courseId,
+                                                question.getId(),
+                                                question.getQuestionType(),
+                                                "alice.b.tmms@gmail.tmt",
+                                                null,
+                                                "alice.b.tmms@gmail.tmt",
+                                                null,
+                                                null);
+        BackDoor.createFeedbackResponse(fra);
+        feedbackEditPage.reloadPage();
+        feedbackEditPage.isAlertClassEnabledForVisibilityOptions(1);
+
+        ______TS("contrib: saving just after edit");
+        feedbackEditPage.clickEditQuestionButton(1);
+        feedbackEditPage.clickSaveExistingQuestionButton(1);
+        feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_QUESTION_EDITED);
+        feedbackEditPage.isAlertClassEnabledForVisibilityOptions(1);
```

Comment:
Is this necessary?

Reply:
I used `isAlertClassEnabledForVisibilityOptions` here to check that the responses are not actually deleted.

If you remember, @whipermr5 pointed out a bug that saving a contribution question just after clicking the editing button, was prompting the response deletion modal. I fixed it and this test is to cover that behavior.